File: trip-kit-booking/src/main/java/com/skedgo/android/tripkit/booking/mybookings/MyBookingsApi.java
package com.skedgo.android.tripkit.booking.mybookings;

import retrofit2.http.GET;
import retrofit2.http.Query;
import io.reactivex.Observable;

public interface MyBookingsApi {
  @GET("booking")
  Observable<MyBookingsResponse> fetchMyBookingsAsync(
      @Query("first") int first,
      @Query("max") int pageSize,
      @Query("bsb") int bsb
  );
}

File: trip-kit-booking/src/main/java/com/skedgo/android/tripkit/booking/mybookings/MyBookingsApi.kt
package com.skedgo.android.tripkit.booking.mybookings

import retrofit2.http.GET
import retrofit2.http.Query
import rx.Observable

interface MyBookingsApi {
  @GET("booking")
  fun fetchMyBookingsAsync(
      @Query("first") first: Int,
      @Query("max") pageSize: Int,
      @Query("bsb") bsb: Int
  ): Observable<MyBookingsResponse>
}